### DISCUSSION TITLE
Is there a way to specify what I want to run on the container startup?

### DISCUSSION THREAD

**Discussion by PassiveLemon** [itzg/docker-minecraft-server#2145] [Q&A] [answered]
I can't seem to find a direct answer. It appears that it runs the server jar file by default but one of my modpacks wants to use its own start.sh. If I use this container for it, I get tons of errors from OOM but running the start.sh independently seems to work fine. I mean this in the sense of an environment variable where I can specify it to run "start.sh" in the data directory rather than whatever the container does by default.

**Comment by itzg** [comment, marked as answer]
The closest to that is `TYPE=CURSEFORGE`, [described here](https://github.com/itzg/docker-minecraft-server#old-approach), where it will unzip the modpack and locate one of several start script names including `start.sh`. Keep in mind that approach is not actively supported since every modpack has its own script behavior and works against the automation this image provides.
